magefile.go

```diff
@@ -32,13 +37,32 @@ var Aliases = map[string]interface{}{
 	"lint": mage.Linter.All,
 }
 
+func Package() {
+	devtools.Build(devtools.DefaultGolangCrossBuildArgs())
+	// This seems like it shouldn't be needed, but
+	// the build tooling for beats wants this,
+	// And for now agent is treating it like a beat
+	devtools.SetBuildVariableSources(
+		&devtools.BuildVariableSources{
+			BeatVersion: "./version/version.go",
+			GoVersion:   ".go-version",
+			DocBranch:   "./docs/version.asciidoc",
```

Comment:
I wouldn't think we need a doc version, because there are no docs here. 

Do we need this with V2 as well? If so we should just modify the tooling to make this optional.

Reply:
These are all required by the `Package` target, and it looks like we blindly copied them all into `elastic-agent` from beats. Currently unsure if it's worth it to make our own `mage package` target separate from beats and agent in the scope of this, since the build system is kind of a snarl. The good news is that the V2 build system removes most of this cruft, so I'm kind of mentally treating this as a temporary holdover until then.